src/main.c

```diff
@@ -15,6 +15,13 @@
 #include <shellapi.h>
 #include <wchar.h>
 #include <wtypes.h>
+
+#ifdef HAVE__ISATTY
```

Comment:
This is not the same behavior as what you removed below. Below, if both are defined we replace isatty with _isatty even ifndef WIN32, but this is just for WIN32. Is that intentional?